test/DataGrid.spec.tsx

```diff
@@ -130,6 +135,38 @@ describe('<DataGrid />', () => {
     });
   });
 
+  describe('When handlePager() is called', () => {
+    let dataGrid;
+
+    before( () => {
+      mock.reset();
+      mock.onPost('url', { ...simpleRequest }).reply(200, {
+        ...simpleRecordsExpected
+      });
+      mock.onPost('url', { ...page2Request }).reply(200, {
+        ...page2Expected
+      });
+
+      dataGrid = (
+        <DataGrid
+          onError={(x: any) => x}
+          gridName='Motorhead'
+          rowsPerPage={10}
+          dataSource={new RemoteDataSource('url', simpleColumnsSample)}
+        />
+      );
+    });
+
+    it('Should refresh the DataGrid DataStream', (done) => {
```

Comment:
I think you should check that the rows in the body actually changed. Look at this test and you will see that this is almost the same as the one for the RemoteDataSource test. But here you're not testing the data source, you are testing that in fact the proper rows are being rendered